<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhitePricesEditor</class>
 <widget class="QDialog" name="WhitePricesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="top_layout">
      <item>
       <widget class="QFrame" name="prices_frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="prices_layout">
         <item>
          <widget class="QFrame" name="retail_frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="retail_layout">
            <item row="0" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>баз.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="rub1_label">
              <property name="text">
               <string>руб.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="retail_spinbox">
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="2">
             <widget class="QLabel" name="retail_label">
              <property name="text">
               <string>Розн.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>изм.</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="rub1_label_2">
              <property name="text">
               <string>руб.</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="retail_base">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="whole_frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="whole_layout">
            <item row="0" column="0" rowspan="2">
             <widget class="QLabel" name="retail_label_2">
              <property name="text">
               <string>Опт.</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>баз.</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="whole_base">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="rub1_label_3">
              <property name="text">
               <string>руб.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>изм.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="whole_spinbox">
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="rub1_label_4">
              <property name="text">
               <string>руб.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="whole_begin_label">
             <property name="text">
              <string>Опт с</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="begin_spinbox">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximum">
              <number>105</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="percents_frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="percents_layout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="percents_checkbox">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="reg_layout">
           <item>
            <layout class="QVBoxLayout" name="plusminus_layout">
             <item>
              <widget class="QRadioButton" name="plus_button">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="minus_button">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>–</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSpinBox" name="percents_spinbox">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="percets_layout">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="reason_layout">
      <item>
       <widget class="QLabel" name="reason_label">
        <property name="text">
         <string>Основание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="reason_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="buttons_frame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="bottom_layout">
       <item row="0" column="0">
        <widget class="QPushButton" name="up_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/up</normaloff>:/up</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="up_write_button">
         <property name="text">
          <string>Записать</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/up</normaloff>:/up</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QPushButton" name="close_button">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="down_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/down</normaloff>:/down</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="down_write_button">
         <property name="text">
          <string>Записать</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/down</normaloff>:/down</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4"/>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
